### ISSUE TITLE
`impt build run --all` is broken

### ISSUE THREAD

**Issue by deldrid1** [EatonGMBD/imp-central-impt#2]
```bash
$ impt build run --all
/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/yargs.js:1148
      else throw err
           ^
 
TypeError: Cannot read property 'indexOf' of undefined
    at new AuthConfig (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/AuthConfig.js:110:53)
    at Function.getEntity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/AuthConfig.js:88:38)
    at new ImpCentralApiHelper (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/ImpCentralApiHelper.js:52:39)
    at Function.getEntity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/ImpCentralApiHelper.js:46:43)
    at new Entity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/Entity.js:57:44)
    at new Build (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/Build.js:59:1)
    at Object.exports.handler (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/bin/cmds/build/run.js:85:5)
    at Object.runCommand (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/lib/command.js:238:44)
    at Object.parseArgs [as _parseArgs] (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/yargs.js:1063:30)
    at Object.runCommand (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/lib/command.js:198:45)
/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/yargs.js:1148
      else throw err
           ^
 
TypeError: Cannot read property 'indexOf' of undefined
    at new AuthConfig (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/AuthConfig.js:110:53)
    at Function.getEntity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/AuthConfig.js:88:38)
    at new ImpCentralApiHelper (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/ImpCentralApiHelper.js:52:39)
    at Function.getEntity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/ImpCentralApiHelper.js:46:43)
    at new Entity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/Entity.js:57:44)
    at new Build (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/Build.js:59:1)
    at Object.exports.handler (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/bin/cmds/build/run.js:85:5)
    at Object.runCommand (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/lib/command.js:238:44)
    at Object.parseArgs [as _parseArgs] (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/yargs.js:1063:30)
    at Object.runCommand (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/lib/command.js:198:45)
/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/yargs.js:1148
      else throw err
           ^
 
TypeError: Cannot read property 'indexOf' of undefined
    at new AuthConfig (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/AuthConfig.js:110:53)
    at Function.getEntity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/AuthConfig.js:88:38)
    at new ImpCentralApiHelper (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/ImpCentralApiHelper.js:52:39)
    at Function.getEntity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/ImpCentralApiHelper.js:46:43)
    at new Entity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/Entity.js:57:44)
    at new Build (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/Build.js:59:1)
    at Object.exports.handler (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/bin/cmds/build/run.js:85:5)
    at Object.runCommand (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/lib/command.js:238:44)
    at Object.parseArgs [as _parseArgs] (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/yargs.js:1063:30)
    at Object.runCommand (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/lib/command.js:198:45)
/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/yargs.js:1148
      else throw err
           ^
 
TypeError: Cannot read property 'indexOf' of undefined
    at new AuthConfig (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/AuthConfig.js:110:53)
    at Function.getEntity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/AuthConfig.js:88:38)
    at new ImpCentralApiHelper (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/ImpCentralApiHelper.js:52:39)
    at Function.getEntity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/ImpCentralApiHelper.js:46:43)
    at new Entity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/Entity.js:57:44)
    at new Build (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/Build.js:59:1)
    at Object.exports.handler (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/bin/cmds/build/run.js:85:5)
    at Object.runCommand (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/lib/command.js:238:44)
    at Object.parseArgs [as _parseArgs] (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/yargs.js:1063:30)
    at Object.runCommand (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/lib/command.js:198:45)
/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/yargs.js:1148
      else throw err
           ^
 
TypeError: Cannot read property 'indexOf' of undefined
    at new AuthConfig (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/AuthConfig.js:110:53)
    at Function.getEntity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/AuthConfig.js:88:38)
    at new ImpCentralApiHelper (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/ImpCentralApiHelper.js:52:39)
    at Function.getEntity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/ImpCentralApiHelper.js:46:43)
    at new Entity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/Entity.js:57:44)
    at new Build (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/Build.js:59:1)
    at Object.exports.handler (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/bin/cmds/build/run.js:85:5)
    at Object.runCommand (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/lib/command.js:238:44)
    at Object.parseArgs [as _parseArgs] (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/yargs.js:1063:30)
    at Object.runCommand (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/lib/command.js:198:45)
/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/yargs.js:1148
      else throw err
           ^
 
TypeError: Cannot read property 'indexOf' of undefined
    at new AuthConfig (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/AuthConfig.js:110:53)
    at Function.getEntity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/AuthConfig.js:88:38)
    at new ImpCentralApiHelper (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/ImpCentralApiHelper.js:52:39)
    at Function.getEntity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/ImpCentralApiHelper.js:46:43)
    at new Entity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/Entity.js:57:44)
    at new Build (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/Build.js:59:1)
    at Object.exports.handler (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/bin/cmds/build/run.js:85:5)
    at Object.runCommand (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/lib/command.js:238:44)
    at Object.parseArgs [as _parseArgs] (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/yargs.js:1063:30)
    at Object.runCommand (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/lib/command.js:198:45)
/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/yargs.js:1148
      else throw err
           ^
 
TypeError: Cannot read property 'indexOf' of undefined
    at new AuthConfig (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/AuthConfig.js:110:53)
    at Function.getEntity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/AuthConfig.js:88:38)
    at new ImpCentralApiHelper (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/ImpCentralApiHelper.js:52:39)
    at Function.getEntity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/ImpCentralApiHelper.js:46:43)
    at new Entity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/Entity.js:57:44)
    at new Build (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/Build.js:59:1)
    at Object.exports.handler (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/bin/cmds/build/run.js:85:5)
    at Object.runCommand (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/lib/command.js:238:44)
    at Object.parseArgs [as _parseArgs] (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/yargs.js:1063:30)
    at Object.runCommand (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/lib/command.js:198:45)
/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/yargs.js:1148
      else throw err
           ^
 
TypeError: Cannot read property 'indexOf' of undefined
    at new AuthConfig (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/AuthConfig.js:110:53)
    at Function.getEntity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/AuthConfig.js:88:38)
    at new ImpCentralApiHelper (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/ImpCentralApiHelper.js:52:39)
    at Function.getEntity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/ImpCentralApiHelper.js:46:43)
    at new Entity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/Entity.js:57:44)
    at new Build (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/Build.js:59:1)
    at Object.exports.handler (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/bin/cmds/build/run.js:85:5)
    at Object.runCommand (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/lib/command.js:238:44)
    at Object.parseArgs [as _parseArgs] (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/yargs.js:1063:30)
    at Object.runCommand (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/lib/command.js:198:45)
/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/yargs.js:1148
      else throw err
           ^
 
TypeError: Cannot read property 'indexOf' of undefined
    at new AuthConfig (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/AuthConfig.js:110:53)
    at Function.getEntity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/AuthConfig.js:88:38)
    at new ImpCentralApiHelper (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/ImpCentralApiHelper.js:52:39)
    at Function.getEntity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/ImpCentralApiHelper.js:46:43)
    at new Entity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/Entity.js:57:44)
    at new Build (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/Build.js:59:1)
    at Object.exports.handler (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/bin/cmds/build/run.js:85:5)
    at Object.runCommand (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/lib/command.js:238:44)
    at Object.parseArgs [as _parseArgs] (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/yargs.js:1063:30)
    at Object.runCommand (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/lib/command.js:198:45)
/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/yargs.js:1148
      else throw err
           ^
 
TypeError: Cannot read property 'indexOf' of undefined
    at new AuthConfig (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/AuthConfig.js:110:53)
    at Function.getEntity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/AuthConfig.js:88:38)
    at new ImpCentralApiHelper (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/ImpCentralApiHelper.js:52:39)
    at Function.getEntity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/ImpCentralApiHelper.js:46:43)
    at new Entity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/Entity.js:57:44)
    at new Build (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/Build.js:59:1)
    at Object.exports.handler (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/bin/cmds/build/run.js:85:5)
    at Object.runCommand (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/lib/command.js:238:44)
    at Object.parseArgs [as _parseArgs] (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/yargs.js:1063:30)
    at Object.runCommand (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/lib/command.js:198:45)
/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/yargs.js:1148
      else throw err
           ^
 
TypeError: Cannot read property 'indexOf' of undefined
    at new AuthConfig (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/AuthConfig.js:110:53)
    at Function.getEntity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/AuthConfig.js:88:38)
    at new ImpCentralApiHelper (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/ImpCentralApiHelper.js:52:39)
    at Function.getEntity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/ImpCentralApiHelper.js:46:43)
    at new Entity (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/util/Entity.js:57:44)
    at new Build (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/lib/Build.js:59:1)
    at Object.exports.handler (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/bin/cmds/build/run.js:85:5)
    at Object.runCommand (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/lib/command.js:238:44)
    at Object.parseArgs [as _parseArgs] (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/yargs.js:1063:30)
    at Object.runCommand (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/yargs/lib/command.js:198:45)
```

**Comment by alexey-nbl** [comment]
Fixed (together with several other fixes). PR will come soon...

**Comment by mm-gmbd** [comment]
Fixed with #12
